pom.xml

```diff
@@ -212,7 +212,7 @@
                 <artifactId>netty-handler</artifactId>
                 <version>4.1.107.Final</version>
             </dependency>
-            
+
```

Comment:
Please revert this whitespace only change, it is polluting the Git attribution.

Reply:
done